sdk/eventhub/Azure.Messaging.EventHubs.Processor/perf/Azure.Messaging.EventHubs.Processor.Perf.sln

```diff
@@ -15,6 +15,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Test.Perf", "..\..\..\..\common\Perf\Azure.Test.Perf\Azure.Test.Perf.csproj", "{1F5EE4F0-7DE8-4E3E-819C-58F34D17C2A1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.Core", "..\..\..\core\Azure.Core\src\Azure.Core.csproj", "{01AA6564-166A-411E-9B0D-6A2CABA246F7}"
```

Comment:
Do you mind deleting `Azure.Messaging.EventHubs.Perf.sln`?   It's no longer needed and I missed doing so when I moved to the [filter set](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/perf/Azure.Messaging.EventHubs.Perf.slnf) the other day.

Reply:
Sorry, this PR is large with a ton of checks. Don't want to kick them all again.